react/package.json

```diff
@@ -9,7 +9,7 @@
     "classnames": "^2.2.6",
     "graphql": "^14.0.2",
     "jsdom": "^11.8.0",
-    "react-apollo": "^2.1.3",
+    "react-apollo": "^2.3.2",
```

Comment:
Not sure if this is necessary. Anyway, yarn.lock is already resolving ^2.1.3 to 2.3.2.